eel_hole/templates/404.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
+
+{% block title %}PUDL Viewer - page not found{% endblock %}
+
+{% block content %}
+
+<div class="section">
+  <div class="container is-max-desktop">
+    <div class="block title">
+    Page not found
+    </div>
+    <div class="content">
+      <p>
+        Uh oh! The URL you requested wasn't found. Sorry!
+      </p>
+      <p>
+        We're in the process of migrating our old data portal website to this one, so this may be a temporary situation.
+      </p>
+      <p>
+        In any case, you may want to <span class="link"><a href="/">go back to the home page</a></span>.
```

Comment:
Mystified by the need for span class link but it was clearly solving some problem -- what bad thing does it do without the span?

Reply:
Hmm, I actually didn't test this without the span. I had thrown it in for some `bulma` styling reasons but let me check if the link renders nicely without.

Reply:
It works fine. Ripping it out.